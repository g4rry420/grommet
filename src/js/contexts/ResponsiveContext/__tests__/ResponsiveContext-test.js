import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';
import 'regenerator-runtime/runtime';

import { Grommet } from '../../../components/Grommet';
import { ResponsiveContext } from '..';

describe('ResponsiveContext', () => {
  test('basic', () => {
    const { container } = render(
      <Grommet>
        <ResponsiveContext.Consumer>{size => size}</ResponsiveContext.Consumer>
      </Grommet>,
    );
    const tree = container.firstChild;
    expect(tree).toMatchSnapshot();
  });
});
