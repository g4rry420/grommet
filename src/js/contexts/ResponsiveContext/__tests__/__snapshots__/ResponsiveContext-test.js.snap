// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveContext basic 1`] = `
.c0 {
  font-size: 18px;
  line-height: 24px;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

<div
  class="c0"
>
  small
</div>
`;
